### PR TITLE
Lab Command

### PR DESCRIPTION
**Pull request by DeanMiles**
Alan Franas
215717

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/commands/ComplexCommand.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package edu.kis.powp.jobs2d.commands;
+
+import java.util.ArrayList;
+
+public class ComplexCommand implements DriverCommand {
+
+    private ArrayList<DriverCommand> commands = new ArrayList<>();
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/commands/ComplexCommand.java:7]: List type (attribute and in the constructor) would be sufficient and would not limit the implementation.

File: src/main/java/edu/kis/powp/jobs2d/commands/ComplexCommand.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    public ComplexCommand() {
+    }
+
+    public ComplexCommand(ArrayList<DriverCommand> commands) {
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/commands/ComplexCommand.java:12]: also List

**Review by kowallus — COMMENTED** [review]
4 pts (too late -1 pts)

**Comment by DeanMiles** [comment]
> 4 pts (too late -1 pts)
@kowallus 
command

Hello, I do not undestand why I have lost point If the information on Wikamp says that I did it on time. 
Best regards,
Alan Franas

Edit. I also did not found information about other deadline in group chat at Teams.

**Pull request closed without merging**